amnestywebsite/humanity-theme
# Adds the chunky heading with links block pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/171

**Steps to test**:
1. Edit a post
2. Open the block inserter
3. Switch to patterns tab
4. Check for the chunky heading with links
5. Insert it and check it matches the example from the ticket

Example: http://bigbite.im/v/fT4KWn
